README.md

```diff
@@ -162,10 +162,33 @@ Either `field.value` or `field.value.pattern` must be defined to apply filter.
 
 Only, `string`, `numeric` and `boolean` types are considered for matching purposes, other types are ignored.
 
+### `ExtractTopicFromValueSchema`
+
+This transformation checks the record value schema name and if it exists uses it as the topic name.
+
+- `io.aiven.kafka.connect.transforms.ExtractTopicFromValueSchema$Name` - works on value schema name.
+
+By default the transformation uses the content of the schema.name field of the record value.
```

Comment:
```suggestion
By default the transformation uses the content of the schema.name field of the record value and uses the first matched group.
```

This would imply changing the example regex to give single matched group. The current regex gives to in Java.